### PR TITLE
Add settings option to forcibly change language to English

### PR DESCRIPTION
**Pull request by prattpratt**

### CODE REVIEW

**Comment by maks** [comment]
Thank you for contributing this, I'll try to have a look through it soon.

**Comment by prattpratt** [comment]
@maks this is really tiny PR. So it does not take much time to review.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -208,4 +208,6 @@
     <string name="error_no_worktree">This is a bare repository!</string>
     <string name="error_invalid_index">The Git index is invalid!</string>
     <string name="error_file_not_found">File not found!</string>
+    <string name="preference_eng_lang">English Language</string>
```
Comment (maks) [inline app/src/main/res/values/strings.xml:211]: please name this to more clearly indicate its purpose, eg "Use English to override device language"

File: app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java [hunk 2]
```diff
@@ -46,6 +50,23 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         BasicFunctions.setActiveActivity(this);
         setTheme(Profile.getThemeResource(getApplicationContext()));
+        updateLocale(Profile.isEngLang(getApplicationContext()));
+    }
+
+    private void updateLocale(boolean isEngLang) {
```
Comment (maks) [inline app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java:56]: pls see other comment regarding parameter naming

**Review by maks — CHANGES_REQUESTED** [review]
Thank you for contributing this and apologies for the delay in reviewing. Looks good, just a few comments regarding naming.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (prattpratt) [reply app/src/main/res/values/strings.xml:211]: This is title, it should be short, but Ive added more clear summary.

File: app/src/main/java/me/sheimi/android/utils/Profile.java [hunk 3]
```diff
@@ -75,4 +75,9 @@ public static int getStyledResource(Context context, int unstyled) {
         a.recycle();
         return styled;
     }
+
+    public static boolean isEngLang(Context context) {
```
[the remark this answers is not in the document]
Comment (prattpratt) [reply app/src/main/java/me/sheimi/android/utils/Profile.java]: Done

File: app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java (hunk 2 above)
Comment (prattpratt) [reply app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java]: Done

**Comment by maks** [comment]
Thank you for contributing this and apologies for taking so long to this in.

**Pull request merged**